docs/guides/fetch-data/01-read-profile-data.md

```diff
@@ -0,0 +1,527 @@
+---
+sidebar_label: 'Read Profile Data'
+sidebar_position: 3.1
+---
+
+# Read Universal Profile Data
+
+In this guide, we will learn how to:
+
+- call a Universal Provile address
+- read data from a Universal Profile
+
+![Universal Profile example on universalprofile.cloud](./img/example-up.png)
+
+We will use:
+
+- [web3.js](https://web3js.readthedocs.io/en/v1.7.0/) for utility as well as connecting with the LUKSO network
+- [erc725.js](../../tools/erc725js/getting-started/) library to check the interface of an profile.
+- [isomorphic-fetch](https://github.com/matthew-andrews/isomorphic-fetch) to enable you to use fetch in plain Node code
+
+## Table of Contents
+
+1. Check for valid Blockchain Address
+2. Call the Universal Profile
+3. Read the Dataset
+   1. Fetch the profile's metadata
+   2. Fetch the profile's picture properties
+   3. Fetch the profile's universal receiver
+
+## Step 1 - Check for valid Blockchain Address
+
+When receiving data from a Universal Profile within an application, it is commonly used as input. You want to make sure that the address is valid in the first place. You can do a checkup by converting the value into a checksum address from the `web3.js` library. We need to install the library first. You can do this by opening up the terminal in the project's folder of your choice.
+
+```shell
+npm install web3
+```
+
+After installation, we can move on with the JavaScript file. During the process, we will always use the same file. To make the guide more understandable, we use a sample profile address. You will most likely exchange this static variable with a dynamic value from an input field or fetching process within your app.
+
+```javascript title="read_profile.js"
+// Import and Network Setup
+const Web3 = require('web3');
+const web3 = new Web3('https://rpc.l14.lukso.network');
+
+// Our static Universal Profile address
+const sampleProfileAddress = '0x0C03fBa782b07bCf810DEb3b7f0595024A444F4e';
+
+/*
+ * Check if @param is a valid blockchain address
+ *
+ * @returns: boolean
+ */
+function isValidAddress(address) {
+  let formattedAddress = web3.utils.toChecksumAddress(address);
+  return web3.utils.checkAddressChecksum(formattedAddress);
+}
+
+// Debug
+console.log(isValidAddress(sampleProfileAddress));
+```
+
+If the function `isValidAddress` gives us back `true`, the address is valid and can be used.`
+
+## Step 2 - Call the Universal Profile
+
+To inspect the address and check if it has an ERC725 contract, we can call its interface through the `erc725.js` library. The instance of the contract will need the following information:
+
+- [LSP2 - ERC725Y JSON Schema](../../standards/generic-standards/lsp2-json-schema/) which describes contract functions
+
+  - SupportedStandards will fetch the interface
```

Comment:
This is incorrect.

You have also have to give the Metadata Standard to check for, so for instance in this guide, the key would be `SupportedStandards:LSP3UniversalProfile`